docs/eigenda/operator-guides/requirements/system-requirements.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 1
+---
+
+# System Requirements
+
+## General System Requirements
+
+The EigenDA network design dictates that operators with greater stake will
+receive more frequent chunk dispersal requests, since they operate with greater
+cryptoeconomic security. This means that an operator's node requirements are a
+function of the total amount of stake they wield across all quorums, which we
+call 'Total Quorum Stake' (TQS). For example, if an operator Foobar has 3% stake
+on the restaked ETH quorum, and 5% ETH on a staked WETH quorum, then operator
+Foobar's TQS is 8%.
+
+Operators should use the following table to determine which [EigenLayer node class](/eigenlayer/operator-guides/eigenlayer-node-classes#general-purpose-eigenlayer-node-classes)
+is appropriate for their level of stake:
+
+| Total Quorum Stake (TQS) |                         | Supported Throughput |
+| ------------------------ | ----------------------- | -------------------- |
+| Up to 0.03% (Solo staker)      | General Purpose - large | 80 Kbps              |
+| Up to 0.2%                     | General Purpose - xl    | 500 Kbps             |
+| Up to 20%                      | General Purpose - 4xl   | 50 Mbps              |
+
+Professional operators with large or variable amounts of delegated stake should
+select the `4xl` node class. The `large` class is intended to be used by solo
+stakers with the minimal allowed quantity of stake.
+
+We will update this specification to include new EigenLayer node classes as they
+are introduced.
+
+## Node Storage Requirements
+
+EigenDA nodes **must** provision high-performance SSD storage in order to keep
+up with network storage and retrieval tasks. Failure to maintain adequate
+performance will result in unacceptable validation latency and automatic churn.
+
+The following table summarizes required storage capacity based on TQS:
+
+| Total Quorum Stake (TQS) | Allocated Throughout | Required Storage |
```

Comment:
What is "Allocated Throughput"? The numbers seem to be the same as "Supported Throughput"; if they are the same, need to use the same term